python/pyproject.toml

```diff
@@ -1,20 +1,46 @@
 [build-system]
-requires = ["maturin>=1.3.0,<1.4.0", "setuptools", "wheel", "Cython"]
+requires = ["maturin>=1.5.1", "setuptools", "wheel", "Cython"]
 build-backend = "maturin"
 
 [project]
 name = "bystro"
-requires-python = ">=3.10"
+requires-python = ">=3.11.0"
 classifiers = [
     "Programming Language :: Rust",
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
-version = "1.0.0"
+version = "1.0.3"
+dependencies = [
```

Comment:
Does moving the dependencies here fix the error where requirements.txt wasn't being automatically installed?